networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -263,11 +263,18 @@ def allocate_hosting_port(self, context, router_id, port_db, network_type,
         # If this is a router interface, the VLAN comes from APIC.
         # If it's the gateway, the VLAN comes from the segment ID
         if port_db.get('device_owner') == DEVICE_OWNER_ROUTER_GW:
-            return super(AciVLANTrunkingPlugDriver,
-                         self).allocate_hosting_port(
-                             context, router_id,
-                             port_db, network_type, hosting_device_id
-                         )
+            ext_dict, net = self._get_external_network_dict(context, port_db)
```

Comment:
what happens if user forgot to provide the entire section at all? then we will just use the default set then everything will seem fine however this is probably not what user wants....Maybe we should just get rid of the default then?